<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFinder</class>
 <widget class="QMainWindow" name="TextFinder">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextFinder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Votre mot :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="chooseWordLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="findButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Faire une recherche globale du mot</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Faire une recherche mot par mot</string>
        </property>
        <property name="text">
         <string>Suivant</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="previousButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Précédent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fullWordCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Mot complet</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="wordComptLabel">
        <property name="toolTip">
         <string>Indique le nombre de mot(s) trouvé</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="wordNumberLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="currentloadedfile">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="showText">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cursorLabel">
      <property name="toolTip">
       <string>Indique la ligne et la lettre du curseur</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>Remettre la fenetre a zero</string>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecent1_2"/>
    <addaction name="actionRecent2_2"/>
    <addaction name="actionRecent3_2"/>
    <addaction name="actionRecent4"/>
    <addaction name="actionRecent5"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Sauvegarder sous</string>
   </property>
  </action>
  <action name="actionLightDark">
   <property name="text">
    <string>Dark/Light Style</string>
   </property>
  </action>
  <action name="actionWhite_Style">
   <property name="text">
    <string>White Style</string>
   </property>
  </action>
  <action name="actionRecent1">
   <property name="text">
    <string>Recent1</string>
   </property>
  </action>
  <action name="actionRecent2">
   <property name="text">
    <string>Recent2</string>
   </property>
  </action>
  <action name="actionRecent3">
   <property name="text">
    <string>Recent3</string>
   </property>
  </action>
  <action name="actionRecent1_2">
   <property name="text">
    <string>Recent1</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRecent2_2">
   <property name="text">
    <string>Recent2</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRecent3_2">
   <property name="text">
    <string>Recent3</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRecent4">
   <property name="text">
    <string>Recent4</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRecent5">
   <property name="text">
    <string>recent5</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>TextFinder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
